README.md

```diff
@@ -180,10 +180,7 @@ Please follow the platform-specific build instructions in the files below:
 ### Generic build tips
 
 **Qt version compatibility**
-Multipass is tested with **Qt 6.10.0**. Newer patch versions along the 6.10 track (e.g. 6.10.4) should be fine. Newer minor versions may work, but they may cause compatibility issues.
-
-**ARM64 and cross-compilation**
-If you're using Apple Silicon (arm64) or cross-compiling, ensure that your `PATH` and `CMAKE_PREFIX_PATH` environment variables point to the correct Qt installation for your system architecture.
+Multipass is tested with **Qt 6.9.0**. Newer patch versions along the 6.9 track (e.g. 6.9.1) should be fine. Newer minor versions may work, but they may cause compatibility issues.
```

Comment:
Extremely minor nitpick. It looks like we're using [Qt 6.9.1](https://github.com/microsoft/vcpkg/blob/4334d8b4c8916018600212ab4dd4bbdc343065d1/ports/qt/vcpkg.json#L3).

```suggestion
Multipass is tested with **Qt 6.9.1**. Newer patch versions along the 6.9 track (e.g. 6.9.2) should be fine. Newer minor versions may work, but they may cause compatibility issues.
```